index.bs

```diff
@@ -375,6 +375,8 @@ A tone map derived image item ('tmap') as defined in [[!HEIF]] may be used in an
 
 In these sections, a "sample" refers to the value of a pixel for a given channel.
 
+With the Sample Transform file construction, pixels at the same position in multiple input image items can be aggregated into a single output pixel using basic mathematical operations. This makes it possible for [=AVIF=] to support 16 or more bits of precision per sample, while still offering backward compatibility through a regular 8 to 12-bit [=AV1 Image Item=] containing the most significant bits of each sample.
```

Comment:
Nit: change "aggregated" to "combined"

The reason is that "aggregate" means "combine by adding" or "gather into a whole", so it may imply certain mathematical operations such as addition or concatenation. Using a less specific word like "combine" seems better.